app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -337,7 +338,6 @@ struct DeviceDelegateOpenXR::State {
       VRB_LOG("OpenXR runtime supports XR_FB_render_model");
     }
 
-    bool isEyeTrackingSupported { false };
     if (OpenXRExtensions::IsExtensionSupported(XR_EXT_EYE_GAZE_INTERACTION_EXTENSION_NAME)) {
         isEyeTrackingSupported = eyeGazeProperties.supportsEyeGazeInteraction;
```

Comment:
This field is now indicating 2 different things, as described in the commit message; shouldn't we choose a more descriptive name for it ? 

Additionally, aren't we interested on knowing just whether the extension is supported or not  ? Just after this code block we are calling to VRBrowser::SetEyeTrackingSupported(isEyeTrackingSupported), so may send 'false' as value when the extension is really supported. 

Also, in the same line, you are describing in the commit message that we need to perform 2 different tests; the second seems something that the app would need to wait for, at the contrary to the first test, where the extension is supported or not invariably.

Reply:
The main point of the patch is that we were only checking that the runtime declares eye tracking as supported, but we really want to know whether eye tracking is working. If it's working then it's clearly supported, but the opposite is not always true. The problem with the current code is that it's incomplete as it assumes that supporting eye tracking means that is working.

Also Java code does not need to wait for anything, we notify it and it listens to the event. There is no logical guarantee that the native code will notify Java about the eye tracking status before the user is able to open the settings dialog, but it's extremely unlikely as this happens very early in the startup process.